activation/activation.go

```diff
@@ -611,13 +613,13 @@ func (b *Builder) createAtx(ctx context.Context, challenge *types.NIPostChalleng
 
 	atx := types.NewActivationTx(
 		*challenge,
-		&b.nodeID,
```

Comment:
if you had assigned SmesherID inside this function you won't need to change other files....

Reply:
not a request for change. just quibbling :P